data/src/test/assets/api_mocks/feedback_reporting.json

```diff
@@ -30,7 +30,7 @@
   },
   "report_creation_timestamp_ms": 1610519337000,
   "system_context": {
-    "package_version_name": "example.package.name",
+    "platform_version": "1.0-example_version",
```

Comment:
Suggest using: 0.1-alpha-abcdef1234

That more closely matches the app version format.

Reply:
Done.